### DISCUSSION TITLE
Importing as ES Module from CDN

### DISCUSSION THREAD

**Discussion by josephm1** [FranckFreiburger/vue3-sfc-loader#8] [Q&A] [answered]
As you can see from the title , I am having trouble importing vue3-sfc-loader as ES Module from CDN, here is the import statement

```
import {
    loadModule
} from "https://cdn.jsdelivr.net/npm/vue3-sfc-loader/dist/vue3-sfc-loader.js"

````

Here is the error in chrome dev tools; Uncaught SyntaxError: The requested module 'https://cdn.jsdelivr.net/npm/vue3-sfc-loader/dist/vue3-sfc-loader.js' does not provide an export named 'loadModule'

**Comment by FranckFreiburger** [comment, marked as answer]
Hello josephm1,
At the moment, `vue3-sfc-loader.js` is injected in the global namespace only (`const { loadModule } = window['vue3-sfc-loader'];`)

Feel free to open a new feature request for importing vue3-sfc-loader as ES Module.

**Reply by josephm1** [reply]
Thank you so much for getting back to me so quickly, I'll make a feature request now,  and some include additional details.
